virttest/data_dir.py

```diff
@@ -128,7 +115,7 @@ def get_backing_data_dir():
 
 
 def get_root_dir():
-    return ROOT_DIR
+    return os.path.dirname(BASE_BACKEND_DIR)
```

Comment:
The get_root_dir is fairly used in many locations, maybe we should keep the `ROOT_DIR` (or we could adjust the tests in most cases as they usually want shared/data dir instead)